packages/extension-utils/src/APIErrorDisplay/index.ts

```diff
@@ -24,4 +24,9 @@
 
  */
 
-export * from './ExtMarkdown'
+export * from './APIErrorContent'
```

Comment:
would exporting only `APIErrorDisplay` and `APIErrorDialog` suffice here?

Reply:
probably